lib/app.rb

```diff
@@ -58,12 +60,7 @@ def run
           if !@projects || @projects.size == 0
```

Comment:
I think I would pull this out as a method: `no_projects?` to make this flow easier to understand.